README.md

```diff
@@ -104,12 +95,17 @@ The API methods are _mostly_ just wrappers around [`signature_pad`'s API](https:
 
 ## Example
 
+Have a look at the [`example/`](example/) directory.
+
+This example can be run with:
+
 ```bash
-$ npm start
+npm start
 ```
 
-Navigate to [http://localhost:8080/](http://localhost:8080/) in your browser
-and you should be able to see the signature canvas in action.
+Followed by navigating to [http://localhost:8080/](http://localhost:8080/) in your browser and to see the signature canvas in action.
```

Comment:
should probably be lowercase
```suggestion
followed by navigating to [http://localhost:8080/](http://localhost:8080/) in your browser and to see the signature canvas in action.
```

Reply:
the "and" is now a typo after the change as well

Reply:
made it lowercase, removed the "and", and just changed the end to "to view it" as the "in action" portion also make it seem like the only way to view a demo.